neutron/compute.sls

```diff
@@ -129,7 +129,8 @@ remote_ovsdb_access:
 
 enable_overlays:
   cmd.run:
-  - name: "ovs-vsctl set open . external-ids:ovn-encap-type=geneve,vxlan"
+  - name: "ovs-vsctl set open .
+  external-ids:ovn-encap-type={{ compute.backend.ovn_encap_type }}"
```

Comment:
typo? why extra line?

Reply:
Look at whole state file, please - I wrote this in same manner like another options for ovs-ovn, which were also divided into two lines, so in my opinion this construction is correct. Let me know, please..

Reply:
Corrected in last commit in this pull rq. Please review...